.github/workflows/pr-check_markdownlint.yml

```diff
@@ -0,0 +1,38 @@
+name: Markdownlint (PR files)
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "**/*.md"
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v29
```

Comment:
Not a blocker, but this is [up to `v34.4.4` now](https://github.com/tj-actions/changed-files/releases/tag/v34.4.4), there were some bugs we hit often on the content repo that are now fixed in later versions.

Reply:
Updated to the floating `v34`

Reply:
Thank you! Is this equivalent to `v34@latest`? We had issues with `v34.4.2` as far as I remember.

Reply:
OK understood, thanks 👍🏻